doc/sphinx/source/getting_started/install.rst

```diff
@@ -124,6 +124,8 @@ code (called ESMValTool if you did not choose a different name) and run
 
     conda env create --name esmvaltool --file environment.yml
 
+This installs ESMValCore as a dependency. Developers might want to
```

Comment:
I would remove the note about installing ESMValCore from source, this is not needed in a getting started guide and something that most users and diagnostic developers will probably not need.

Reply:
I think ideally the whole section about installing ESMValTool from source should be gone from the getting started guide, or we should even have a separate main menu item **Installation** as many documentations do have this and there more advanced installations make sense, as well as the remark that an source installation of esmvalcore is possible.

Reply:
Yes, installation from source should be moved to become the first chapter here doc/sphinx/source/esmvaldiag/

Reply:
I agree, but restructuring the documentation is beyond the scope of this PR.

Reply:
I agree, but (as discussed) restructuring the entire documentation is beyond the scope of this PR.

Reply:
> I agree, but (as discussed) restructuring the entire documentation is beyond the scope of this PR.

That's fine, but then please address my original comment

Reply:
That note should be gone now.